mysql-test/suite/galera/t/galera_gcache_pagestore.test

```diff
@@ -0,0 +1,331 @@
+#
```

Comment:
Please create a doc bug and list the updated semantics. Ask them to update the existing doc (if any) or create new one. Review it before they publish.